nupic/algorithms/anomaly.py

```diff
@@ -22,34 +22,43 @@
 """Anomaly-related algorithms."""
 
 import numpy
+import collections
 
 from nupic.algorithms.anomaly_likelihood import AnomalyLikelihood
 from nupic.utils import MovingAverage
 
 
 def computeRawAnomalyScore(activeColumns, prevPredictedColumns):
-  """Computes the raw anomaly score.
+    """Computes the raw anomaly score.
```

Comment:
The indentation was changed to 4 spaces for the body of this function but it should be left at 2